dc-infrastructure.tf

```diff
@@ -13,7 +13,10 @@ module "base-infrastructure" {
   eks_additional_roles = var.eks_additional_roles
   whitelist_cidr       = var.whitelist_cidr
 
-  enable_ssh_tcp = local.install_bitbucket
+  enable_ssh_tcp       = local.install_bitbucket
+  resource_tags        = var.resource_tags
```

Comment:
I suppose you were intended to pass it into launch template? It's never get used now. 😂

Reply:
Yup, thanks for noticing! Too many vars...